packages/web/pages/docs/index.mdx

```diff
@@ -83,6 +81,8 @@ await sendMail({
 
 This will send an email with the subject "My First Email" to the `to`, `cc`, and `bcc` addresses. The `component` prop is the React component that will be rendered to HTML for the email.
 
+More info on sending email and configuring your transport can be found in the [Sending email doc](docs/sending-email).
```

Comment:
i didn't expect this to open a new tab, is that the desired behavior?

Reply:
ah no that url is wrong, will fix